src/NodaTime/Text/AnnualDateTypeConverter.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright 2018 The Noda Time Authors. All rights reserved.
+// Use of this source code is governed by the Apache License 2.0,
+// as found in the LICENSE.txt file.
+
+namespace NodaTime.Text
```

Comment:
Given the simplicity of all of these converters, I'd be happy to have a single `TypeConverters.cs` file that has them all in. We don't even need XML doc for them, given their triviality.